linux_os/guide/system/network/networkmanager/networkmanager_dns_mode/ansible/shared.yml

```
@@ -6,7 +6,7 @@
 
 {{{ ansible_instantiate_variables("var_networkmanager_dns_mode") }}}
 
-{{{ ansible_ini_file_set("/etc/NetworkManager/NetworkManager.conf", "main", "dns", "{{ var_networkmanager_dns_mode }}") }}}
+{{{ ansible_ini_file_set("/etc/NetworkManager/NetworkManager.conf", "main", "dns", "{{ var_networkmanager_dns_mode }}", true) }}}
```

I'm sorry I was wrong, this isn't sufficient, they still fail. We need also to update the macro here.

```suggestion
{{{ ansible_ini_file_set("/etc/NetworkManager/NetworkManager.conf", "main", "dns", "{{ var_networkmanager_dns_mode }}", description=None, ignore_spaces=True) }}}

```